vivo-project/Vitro
# Change PropertyKey class name and namespace

**[VIVO GitHub issue 3862](https://github.com/vivo-project/VIVO/issues/3862)**:

# What does this pull request do?
Changes PropertyKey class to UILabel to align with naming already in use, and changes /core/properties/ part of namespace to /vitro/ui-label/ to clearly differentiate this application ontology from the VIVO core ontology.

# How should this be tested?
A description of what steps someone could take to:
* Deploy and confirm that UI labels appear normally in Vitro/Wilma/Tenderfoot.

# Interested parties
@VIVO-project/vivo-committers
